src/sentry/replays/query.py

```diff
@@ -221,7 +223,9 @@ def query_replays_dataset_tagkey_values(
         aggregated_column = Function("identity", parameters=[grouped_column], alias="tag_value")
 
     if tag_like:
-        where.append(Condition(grouped_column, Op.LIKE, tag_like))
+        where.append(
+            Condition(Function("lower", parameters=[grouped_column]), Op.LIKE, tag_like.lower())
```

Comment:
`ILIKE` is case insensitive.

Reply:
ILIKE doesn't seem to be a snuba op.. is there another way to use it in a condition? https://github.com/getsentry/sentry/blob/343fd08cd8baaa4de18351d49549da69e0c7e418/.venv/lib/python3.11/site-packages/snuba_sdk/conditions.py#L21-L21

Reply:
Could be a good opportunity to open a PR on the snuba-sdk repo.

Reply:
This is how we do case insensitive search for replays currently: https://github.com/getsentry/sentry/blob/d14d08fcadfc054a3261355db3ff7e2dd7883dcb/src/sentry/replays/lib/new_query/conditions.py#L133-L136

So you can ignore ilike and use `match`.

Reply:
Kk, we could also use positionCaseInsensitive like Tony suggests below https://github.com/getsentry/sentry/pull/73989#discussion_r1670964386